common/macresman.h

```diff
@@ -266,7 +268,7 @@ class MacResManager {
 	 * @param tail Calculate length from the tail
 	 * @return The MD5 checksum of the resource fork
 	 */
-	String computeResForkMD5AsString(uint32 length = 0, bool tail = false) const;
+	String computeResForkMD5AsString(uint32 length = 0, bool tail = false, ProgressUpdateCallback progressUpdateCallback = nullptr, GUI::IntegrityDialog *dialog = nullptr) const;
```

Comment:
This is so wrong. The proper way to write it would be:

```suggestion
	String computeResForkMD5AsString(uint32 length = 0, bool tail = false, ProgressUpdateCallback progressUpdateCallback = nullptr, void *callbackParameter = nullptr) const;
```